document `includeFoxxQueues` attribute and persist it (#8351)
arangodb/arangodb: Documentation/Books/Manual/Administration/MasterSlave/ReplicationApplier.md

*_users*) should be applied. If set to *true*, changes in these collections will be replicated, 
otherwise, they will not be replicated. It is often not necessary to replicate data from system
collections, especially because it may lead to confusion on the slave because the slave needs to 

> have its own system collections in order to start and keep operational. Please note that there is 

Reviewer: ```suggestion
have its own system collections in order to start and keep operational.

{% hint 'warning' %}
```

> a seperate option *includeFoxxQueues* for controlling whether Foxx queue jobs from the system 

Reviewer: ```suggestion
There is a separate option *includeFoxxQueues* for controlling whether Foxx queue jobs from the system 
```
